mev-relay-rs/src/relay.rs

```diff
@@ -448,6 +451,28 @@ impl Relay {
         state.auctions.insert(auction_request, auction_context);
         Ok(())
     }
+
+    fn verify_blinded_block_signature(
+        &self,
+        signed_block: &mut SignedBlindedBeaconBlock,
+    ) -> Result<(), ethereum_consensus::Error> {
+        let proposer = self
+            .validator_registry
+            .get_public_key(signed_block.message().proposer_index())
+            .unwrap();
+        let slot = signed_block.message().slot(); // signed_block.message().slot() + 1 ???;
+        let genesis_validators_root = [0u8; 4].hash_tree_root().map_err(ConsensusError::from)?; // Should `compute_consenus_signing_root()` take Option<&Root> ?
```

Comment:
you'll see `compute_consensus_signing_root` takes a `Root` value and if you go track it down this is the incorrect `genesis_validators_root`

moreover, it is different per network so we will want to pass it in to the `Relay` depending on which network we are working with

I'd just pass it in to an argument to `Relay::new` and then store it on a field of `Inner`

a simple way to get the root would be to use the beacon APIs via [this method](https://github.com/ralexstokes/ethereum-consensus/blob/89d320933a3b836adf46f282847bd819264797a4/beacon-api-client/src/api_client.rs#L130) -- note: there is a `beacon_node` item in `Service::spawn` you can use

Reply:
I wasn't sure what the root should have been.  

Saw in Builder specs ([here](https://github.com/ethereum/builder-specs/blob/main/specs/bellatrix/builder.md#signing)) that it should be `None`.  Thanks for clarifying